Cargo.toml

```diff
@@ -70,7 +70,7 @@ pallet-bridge-parachains = { version = "0.8.0", default-features = false }
 pallet-bridge-relayers = { version = "0.8.0", default-features = false }
 pallet-broker = { version = "0.7.1", default-features = false }
 pallet-child-bounties = { version = "28.0.0", default-features = false }
-pallet-collator-selection = { version = "10.0.0", default-features = false }
+pallet-collator-selection = { version = "10.0.2", default-features = false }
```

Comment:
A bit off topic, but it might be a good idea to add a workflow check that ensures the dependency versions match a specific Polkadot sdk release. We can probably use [psvm](https://github.com/paritytech/psvm) for this.